README.md

```diff
@@ -28,6 +28,30 @@ For Event watcher, see: [the documentation](https://pipecd.dev/docs/user-guide/e
 
 **Optional**:　Require pipectl >= v0.49.3. The comma-separated list of contexts for the event. Format: key=value,key2=value2
 
+### `commit-hash`
+
+**Optional**: Require pipectl >= 0.51.0. The commit hash that triggers the event. This is required if you want to use PipeCD Deployment Trace feature.
+
+### `commit-title`
+
+**Optional**: The title of commit that triggers the event.
+
```

Comment:
[Ask] Are other filelds optional to use deployment trace?

Reply:
Sorry, I missed that `commit-url` is required as well 👀 Updated
ref: https://github.com/pipe-cd/pipecd/blob/master/pkg/app/server/grpcapi/api.go#L834-L836